### ISSUE TITLE
State screen "Load" button not visible but working

### ISSUE THREAD

**Issue by tsteenbakkers** [Cpasjuste/psnes#5]
Not sure under what circumstances this happens but on my switch (5.1.0 Hekate) when i go to the states screen and click an existing state "Load" button is not visible. It is working when i click A and when i press left / right i can see the selection switch targets it is just not visible.

tested on multiple roms with existing states and fresh new roms w/o states.

i will add a photo later.

forgot to mention this is on v1.1 beta

**Comment by Cpasjuste** [comment]
Hi, thanks for the report. I did notice that one time, but was not able to reproduce it yet.. I have an idea of the problem thought, I'll take a closer look.

**Comment by tsteenbakkers** [comment]
I remember that at one point in time I removed a .state file which was in the same folder as my smc file (in order to figure out where states are saved in your code). It did not affect existing states already from psnes (probably an old.state file from.another emulator). Unsure if the button showed before I deleted the file though. Just a long shot. 

If you want me to test something let me know.
